src/amo/components/HomeHeroBanner/styles.scss

```diff
@@ -1,19 +1,160 @@
-@import "~photon-colors/colors";
+/* stylelint-disable indentation */
+$colors: ("blue", #228ae6, #1c7cd6)
+         ("cyan", #15aabf, #1098ad)
+         ("grape", #be4bdb, #ae3ec9)
+         ("indigo", #4c6ef5, #4263eb)
+         ("pink", #e64980, #d6336c)
+         ("violet", #7950f2, #7048e8)
+         ("teal", #12b886, #0ca678);
 
-.HomeHeroBanner {
-  .HeroSection-styleName--Home-featured-extensions {
-    background: $teal-70;
-  }
+/* stylelint-enable indentation */
 
-  .HeroSection-styleName--Home-youtube-high-definition {
-    background: $green-70;
+@function encodeColor($string) { /* stylelint-disable-line */
```

Comment:
it seems `@function` and `@return` (below) are not enabled by default. It is possible to update the stylelint config though: https://github.com/stylelint/stylelint/issues/1612.

Reply:
Yep it would be nice to add that generally - can be a follow-up if that's easier.

Reply:
We had a config file already, so I have just updated it.